module2/homework/insensitivePalindrom/palindrome.cpp

```diff
@@ -0,0 +1,11 @@
+#include "palindrome.hpp"
+
+#include <algorithm>
+
+bool is_palindrome(std::string str) {
+    str.erase(std::remove_if(str.begin(), str.end(),
+                             [](char element) { return !std::isalpha(element); }),
+              str.end());
```

Comment:
ok, nie potrzebujemy const referencji w tym pliku
jak rozdzielamy enterem zmienne wchodzące do funkcji/algorytmu, itp to każda musi być w nowej linijce i na jednej wysokości czyli coś takiego:

```
str.erase(std::remove_if(str.begin(),
                        str.end(),
                        [](char element) { return !std::isalpha(element); }),
                        str.end());
```

Reply:
Done, w sumie to się zastanawiam jak najlepiej to formatować, bo mam ustawione teraz formatowanie przy zapisie i robi mi z automatu takie, może kiedyś Łukasza podpytam